src/main/scala/com/github/unchama/seichiassist/subsystems/mebius/domain/property/MebiusForcedMaterial.scala

```diff
@@ -16,13 +15,36 @@ sealed trait MebiusForcedMaterial {
 object MebiusForcedMaterial {
   import cats.implicits._
 
+  // ダイヤモンド
   case object None extends MebiusForcedMaterial {
     override def allowedAt(level: MebiusLevel): Boolean = true
     override def next: MebiusForcedMaterial = Leather
   }
 
+  // 革
   case object Leather extends MebiusForcedMaterial {
     override def allowedAt(level: MebiusLevel): Boolean = level >= MebiusLevel(30)
+    override def next: MebiusForcedMaterial = Gold
+  }
+
+  // 金
+  case object Gold extends MebiusForcedMaterial {
+    override def allowedAt(level: MebiusLevel): Boolean = level >= MebiusLevel(30)
+
+    override def next: MebiusForcedMaterial = Iron
+  }
+
+  // 鉄
+  case object Iron extends MebiusForcedMaterial {
+    override def allowedAt(level: MebiusLevel): Boolean = level >= MebiusLevel(30)
+
+    override def next: MebiusForcedMaterial = Chain
+  }
+
+  // チェーン
+  case object Chain extends MebiusForcedMaterial {
+    override def allowedAt(level: MebiusLevel): Boolean = level >= MebiusLevel(30)
+
```

Comment:
nit: 下のようにするか、LeatherとNoneも`allowedAt`と`next`関数の間に空行を入れるといいと思います。

```suggestion
    override def next: MebiusForcedMaterial = Iron
  }

  // 鉄
  case object Iron extends MebiusForcedMaterial {
    override def allowedAt(level: MebiusLevel): Boolean = level >= MebiusLevel(30)
    override def next: MebiusForcedMaterial = Chain
  }

  // チェーン
  case object Chain extends MebiusForcedMaterial {
    override def allowedAt(level: MebiusLevel): Boolean = level >= MebiusLevel(30)
```